### PR TITLE
Release info added from the features repo

### PR DESCRIPTION
**Pull request by idvoretskyi**
Kubernetes 1.3-1.9 releases info moved from the https://github.com/kubernetes/features repo (existing release-1.* directories from the features repo will be removed after this PR will be merged).

FYI @pwittrock @calebamiles @grodrigues3 @spiffxp @jdumars

### CODE REVIEW

File: releases/release-1.8/add_release_notes_email_template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Hello ___SIG Name___!
+
+We are starting to collect release notes for the major changes in Kubernetes 1.8.
+A [draft](https://github.com/kubernetes/features/blob/master/release-1.8/release_notes_draft.md) [1]
+has been prepared in the 1.8 release directory but we need your help to populate the changes by
+component which is how we have organized [previous release notes](https://github.com/kubernetes/features/blob/master/release-1.7/release-notes-draft.md) [2].
```
Comment (stp-ip) [inline releases/release-1.8/add_release_notes_email_template.md]: Broken link (relative needed)

File: releases/release-1.6/go-signal-reports/2017-03-16.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+## Incomplete Features
+- Open issues filed against [`kubernetes/features` in 1.6 milestone](https://github.com/kubernetes/features/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20milestone%3Av1.6): 29 (**unchanged from last report**)
+
+### Action Required
+- Feature owners fill out feature template by **TODAY!**
+- Feature owners submit docs PR which references issue in features repo by **TODAY!**
+- Feature owners prepare release notes by **TODAY!**
+- Address etcd upgrade test failures
+- Begin adding human curated release notes to [the release notes draft](https://github.com/kubernetes/features/blob/release-notes-draft-1.6/release-1.6/release-notes-draft.md) by Friday, 17 March 2017 **please commit directly to the `release-notes-draft-1.6` branch so we can comment on the [associated PR](https://github.com/kubernetes/features/pull/203) collectively**
```
Comment (stp-ip) [inline releases/release-1.6/go-signal-reports/2017-03-16.md]: Broken link (relative needed)

File: releases/release-1.6/go-signal-reports/2017-03-20.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  - https://github.com/kubernetes/features/issues/93
+  - https://github.com/kubernetes/features/issues/88
+  - https://github.com/kubernetes/features/issues/43
+
+### Action Required
+- Feature owners fill out feature template by **TODAY!**
+- Feature owners submit docs PR which references issue in features repo by **TODAY!**
+- Feature owners prepare release notes by **TODAY!**
+- Address etcd upgrade test failures
+- Begin adding human curated release notes to [the release notes draft](https://github.com/kubernetes/features/blob/release-notes-draft-1.6/release-1.6/release-notes-draft.md) by **TODAY** **please commit directly to the `release-notes-draft-1.6` branch so we can comment on the [associated PR](https://github.com/kubernetes/features/pull/203) collectively**
```
Comment (stp-ip) [inline releases/release-1.6/go-signal-reports/2017-03-20.md]: Broken link (relative needed)

File: releases/release-1.6/go-signal-reports/2017-03-21.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+### Action Required
+- Address all suite failures *immediately*
+- Pay particular attention to skew test failures
+- Use the new [triage tool](https://storage.googleapis.com/k8s-gubernator/triage/index.html?test=Simple%20pod%20should%20handle%20in-cluster%20config%7CNginx%20should%20conform%20to%20Ingress%20spec%7CPreStop%20should%20call%20prestop%20when%20killing%20a%20pod), thanks SIG Testing and Team/Test-Infra!
+
+## Incomplete Features
+- Open issues filed against [`kubernetes/features` in 1.6 milestone](https://github.com/kubernetes/features/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20milestone%3Av1.6): 29 (**unchanged from last report**)
+- PRs still needing docs
+  - https://github.com/kubernetes/features/issues/43
+- Human curated release notes still incomplete. Specifically very litte curation of [automatically generated release notes](https://github.com/kubernetes/features/blob/release-notes-draft-1.6/release-1.6/release-notes-draft.md#auto-generated-release-notes)
```
Comment (stp-ip) [inline releases/release-1.6/go-signal-reports/2017-03-21.md]: Broken link (relative needed)

File: releases/release-1.6/go-signal-reports/2017-03-21.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+### Action Required
+- Feature owners fill out feature template by **TODAY!**
+- Feature owners submit docs PR which references issue in features repo by **TODAY!**
+- Feature owners prepare release notes by **TODAY!**
+- Address etcd upgrade test failures
+- Finish adding human curated release notes to [the release notes draft](https://github.com/kubernetes/features/blob/release-notes-draft-1.6/release-1.6/release-notes-draft.md) by **TODAY** **please commit directly to the `release-notes-draft-1.6` branch so we can comment on the [associated PR](https://github.com/kubernetes/features/pull/203) collectively**
```
Comment (stp-ip) [inline releases/release-1.6/go-signal-reports/2017-03-21.md]: Broken link (relative needed)

File: releases/release-1.6/go-signal-reports/2017-03-24.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+- https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gke-gci-1.6-gci-1.5-downgrade-cluster
+
+### Action Required
+- Address all suite failures *immediately*
+- Pay particular attention to skew test failures
+- Use the new [triage tool](https://storage.googleapis.com/k8s-gubernator/triage/index.html?test=Simple%20pod%20should%20handle%20in-cluster%20config%7CNginx%20should%20conform%20to%20Ingress%20spec%7CPreStop%20should%20call%20prestop%20when%20killing%20a%20pod), thanks SIG Testing and Team/Test-Infra!
+
+## Incomplete Features
+- Open issues filed against [`kubernetes/features` in 1.6 milestone](https://github.com/kubernetes/features/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20milestone%3Av1.6): 29 (**unchanged from last report**)
+- Human curated release notes still incomplete. Specifically very litte curation of [automatically generated release notes](https://github.com/kubernetes/features/blob/release-notes-draft-1.6/release-1.6/release-notes-draft.md#auto-generated-release-notes)
```
Comment (stp-ip) [inline releases/release-1.6/go-signal-reports/2017-03-24.md]: Broken link (relative needed)

File: releases/release-1.6/go-signal-reports/2017-03-24.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+### Action Required
+- Feature owners fill out feature template by **TODAY!**
+- Feature owners submit docs PR which references issue in features repo by **TODAY!**
+- Feature owners prepare release notes by **TODAY!**
+- Address etcd upgrade test failures
+- Make all final changes to the [the release notes draft](https://github.com/kubernetes/features/blob/master/release-1.6/release-notes-draft.md) by **TODAY!**
```
Comment (stp-ip) [inline releases/release-1.6/go-signal-reports/2017-03-24.md]: Broken link (relative needed)

File: releases/release-1.6/release-1.6.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+- Release 1.6 on March 22 (Wednesday)
+
+
+# Key features
+[Feature tracking spreadsheet
+link](https://docs.google.com/spreadsheets/d/1nspIeRVNjAQHRslHQD1-6gPv99OcYZLMezrBe3Pfhhg/edit#gid=0)
+
+## Notable operational changes
+
+1. Starting in the 1.6 release the [release team](https://github.com/kubernetes/features/blob/master/release-1.6/release_team.md)
```
Comment (stp-ip) [inline releases/release-1.6/release-1.6.md]: Broken link (relative needed)

File: releases/release-1.7/release-1.7.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+- Release 1.7 on June 28 (Wednesday)
+
+
+# Key features
+[Feature tracking spreadsheet
+link](https://docs.google.com/spreadsheets/d/1IJSTd3MHorwUt8i492GQaKKuAFsZppauT4v1LJ91WHY/edit?usp=sharing)
+
+## Notable operational changes
+
+1. Starting in the 1.7 release the [release team](https://github.com/kubernetes/features/blob/master/release-1.7/release_team.md)
```
Comment (stp-ip) [inline releases/release-1.7/release-1.7.md]: Broken link (relative needed)

File: releases/release-1.8/add_release_notes_email_template.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
+Hello ___SIG Name___!
+
+We are starting to collect release notes for the major changes in Kubernetes 1.8.
+A [draft](https://github.com/kubernetes/features/blob/master/release-1.8/release_notes_draft.md) [1]
```
Comment (stp-ip) [inline releases/release-1.8/add_release_notes_email_template.md]: Broken link (relative needed)

File: releases/release-1.8/add_release_notes_email_template.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+has been prepared in the 1.8 release directory but we need your help to populate the changes by
+component which is how we have organized [previous release notes](https://github.com/kubernetes/features/blob/master/release-1.7/release-notes-draft.md) [2].
+We would like to have the draft release notes finished by 08 September 2017 which is one week after code freeze.
+Once your SIG has filled out their section according to the [guidance](https://github.com/kubernetes/community/issues/484) [3],
+provided by Brian Grant, please check off your section in the checklist
+[embedded in the draft](https://github.com/kubernetes/features/blob/master/release-1.8/release_notes_draft.md#checklist-for-sigs-and-release-team) [4].
```
Comment (stp-ip) [inline releases/release-1.8/add_release_notes_email_template.md]: Broken link (relative needed)

File: releases/release-1.8/add_release_notes_email_template.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+there is agreement within the SIG. Please bring any questions to SIG Release on the [mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-release) [5]
+or on [slack](https://kubernetes.slack.com/messages/C56HWQ0TH/) [6].
+
+Thanks!
+
+- The Kubernetes 1.8 Release Team
+
+Links:
+
+1. https://github.com/kubernetes/features/blob/master/release-1.8/release_notes_draft.md
```
Comment (stp-ip) [inline releases/release-1.8/add_release_notes_email_template.md]: Broken link (relative needed)

File: releases/release-1.8/add_release_notes_email_template.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+2. https://github.com/kubernetes/features/blob/master/release-1.7/release-notes-draft.md
```
Comment (stp-ip) [inline releases/release-1.8/add_release_notes_email_template.md]: Broken link (relative needed)

File: releases/release-1.8/add_release_notes_email_template.md [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+3. https://github.com/kubernetes/community/issues/484
+4. https://github.com/kubernetes/features/blob/master/release-1.8/release_notes_draft.md#checklist-for-sigs-and-release-team
```
Comment (stp-ip) [inline releases/release-1.8/add_release_notes_email_template.md]: Broken link (relative needed)

File: releases/release-1.8/release-1.8.md [hunk 15]
```diff
@@ -0,0 +1,91 @@
...
+  * More RCs as needed to verify release-blocking fixes.
+- Release 1.8 on Sept 27 (Wednesday)
+
+
+# Key features
+[Feature tracking spreadsheet
+link](https://docs.google.com/spreadsheets/d/1AFksRDgAt6BGA3OjRNIiO3IyKmA-GU7CXaxbihy48ns/edit#gid=0)
+
+## Operational Changes 
+1. For the 1.8 release, the [release team](https://github.com/kubernetes/features/blob/master/release-1.8/release_team.md)
```
Comment (stp-ip) [inline releases/release-1.8/release-1.8.md]: Broken link (relative needed)

**Comment by stp-ip** [comment]
Problem in general is that yes it's something specific to the release team and sig, but it's also "user" facing information and moving it from the features repo, which is already sorta hard to find, might make this even harder. Considering that we have official release notes, it's just something to take into consideration.

**Comment by stp-ip** [comment]
Also https://github.com/kubernetes/features/blob/master/EXCEPTIONS.md must be changed to point to the latest release and the correct repo or needs to be removed as it (the link to the release) doesn't seem up to date anyway.

**Comment by idvoretskyi** [comment]
> Problem in general is that yes it's something specific to the release team and sig, but it's also "user" facing information and moving it from the features repo, which is already sorta hard to find, might make this even harder. Considering that we have official release notes, it's just something to take into consideration.

@stp-ip features repo is not the best place to track and store release-related information as it has to track features efforts only. Since now we have the sig-release repo, it could be a better place to store these aftifacts.

Also, some related data was already migrated https://github.com/kubernetes/sig-release/pull/42

**Comment by stp-ip** [comment]
Great. Happy to lgtm, after the comments are addressed with EXCEPTIONS.md being doable in a follow-up PR.

**Comment by idvoretskyi** [comment]
@stp-ip links updated. Thanks for noticing!

**Comment by stp-ip** [comment]
Sidenote: relative links would also work within the same repo.

/ok-to-test
/lgtm

**Pull request merged**